<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="12" column="1">
     <widget class="QPushButton" name="pushButton_r2">
      <property name="text">
       <string>960x540</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QPushButton" name="pushButton_r6">
      <property name="text">
       <string>160x120</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_sendPicture">
      <property name="text">
       <string>SendPicture</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="pushButton_r1">
      <property name="text">
       <string>1280x720</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QPushButton" name="pushButton_r4">
      <property name="text">
       <string>640x360</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="14" column="0">
     <widget class="QPushButton" name="pushButton_r5">
      <property name="text">
       <string>320x240</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QPushButton" name="pushButton_r3">
      <property name="text">
       <string>640x480</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="0">
       <widget class="QPushButton" name="pushButton_send">
        <property name="text">
         <string>Send this:</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_message"/>
      </item>
      <item row="7" column="1">
       <widget class="QTextEdit" name="textEdit_log"/>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="text">
         <string>25</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="inputMask">
         <string extracomment="Input your name"/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>not selected</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_dialogWith">
        <property name="text">
         <string>connect to:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>image quality (0-100)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_connect">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 255);</string>
          </property>
          <property name="text">
           <string>Connect to server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_disconnect">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>Disconnect from server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_status">
          <property name="maximumSize">
           <size>
            <width>9999</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string>status</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="inputMask">
         <string extracomment="Who you want to connect to"/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>your name:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="pushButton_apply">
        <property name="text">
         <string>apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButtonGetUsers">
      <property name="text">
       <string>Get All users</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="img_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionchange_resolution">
   <property name="text">
    <string>change resolution</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
